src/Microsoft.DotNet.ApiSymbolExtensions/Filtering/AccessibilitySymbolFilter.cs

```diff
@@ -10,22 +10,22 @@ namespace Microsoft.DotNet.ApiSymbolExtensions.Filtering
     /// </summary>
     public class AccessibilitySymbolFilter : ISymbolFilter
     {
-        private readonly bool _includeInternalSymbols;
-        private readonly bool _includeEffectivelyPrivateSymbols;
-        private readonly bool _includeExplicitInterfaceImplementationSymbols;
+        public bool IncludeExplicitInterfaceImplementationSymbols { get; }
+        public bool IncludeInternalSymbols { get; }
+        public bool IncludeEffectivelyPrivateSymbols { get; }
```

Comment:
As this is public API, please add triple slash XML documentation for each property.

Reply:
updated